paddle/operators/roi_pool_op.cc

```diff
@@ -0,0 +1,126 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. */
+
+#include "paddle/operators/roi_pool_op.h"
+
+namespace paddle {
+namespace operators {
+
+class RoiPoolOp : public framework::OperatorWithKernel {
```

Comment:
I understand that in many places of our codebase, we haven't strictly followed the [English writing rule](http://www.getitwriteonline.com/archive/011309capitalizationinsentences.htm) that acronyms must be capitalized, but let us start doing so from here by changing `Roi` into `ROI`. Thanks!